bundles/org.openhab.binding.guntamatic/src/main/java/org/openhab/binding/guntamatic/internal/GuntamaticHandler.java

```diff
@@ -436,13 +453,13 @@ private String toLowerCamelCase(String input) {
                     String response = new String(contentResponse.getContent(), Charset.forName(config.encoding));
                     if (url.equals(DAQEXTDESC_URL)) {
                         parseAndJsonInit(response);
-                    } else if (url.equals(DAQDATA_URL)) {
-                        parseAndUpdate(response);
                     } else if (url.equals(DAQDESC_URL)) {
                         parseAndInit(response);
+                    } else if (url.equals(DAQDATA_URL)) {
+                        parseAndUpdate(response);
                     } else {
-                        logger.debug(req);
-                        // PARSET_URL via return
+                        logger.debug("parset request: {}", req);
```

Comment:
typo?
```suggestion
                        logger.debug("parsed request: {}", req);
```

Reply:
no, the term comes from "parameter-set" - the URI which is used to write to the heating system